tests/components/blueprint/test_websocket_api.py

```diff
@@ -622,3 +622,41 @@ async def test_substituting_blueprint_inputs_incomplete_input_2(
         "code": "unknown_error",
         "message": "No substitution found for input blah",
     }
+
+
+async def test_get_blueprint(
+    hass: HomeAssistant, hass_ws_client: WebSocketGenerator
+) -> None:
+    """Test getting the details of a blueprint that does exist."""
+
+    client = await hass_ws_client(hass)
+    await client.send_json_auto_id(
+        {
+            "type": "blueprint/get",
+            "domain": "automation",
+            "path": "test_event_service.yaml",
+        }
+    )
+
+    msg = await client.receive_json()
+
+    assert msg["success"]
+    blueprints = msg["result"]
+    assert blueprints == {
```

Comment:
This is hard to read. Add a constant at module level:
```py
_EXPECTED_BLUEPRINT_YAML = """blueprint:
  name: Call service based on event
  domain: automation
  input:
    trigger_event:
      selector:
        text:
          multiline: false
          multiple: false
    service_to_call:
    a_number:
      selector:
        number:
          mode: box
          step: 1.0
triggers:
  trigger: event
  event_type: !input trigger_event
actions:
  service: !input service_to_call
  entity_id: light.kitchen
"""
```

And change the comparison here to:
```py
    assert blueprints == {"yaml": _EXPECTED_BLUEPRINT_YAML}
```